framework/src/modules/chain/blocks/blocks.js

```diff
@@ -580,7 +581,7 @@ class Blocks extends EventEmitter {
 				this._lastBlock = block;
 				this.logger.info(
 					{ blockId: block.id, height: block.height },
-					'Rebuilding block',
+					'Rebuilt block',
```

Comment:
Hmmm, the function name and log name is different, it would be nice if could make them inline.